    Blocking waiting for file lock on build directory
    Checking google-cloudtasks2 v0.1.0 (/Users/sthiel/dev/google-apis/generated/gen/cloudtasks/v2/lib)
error[E0433]: failed to resolve: unresolved import
    --> gen/cloudtasks/v2/lib/src/lib.rs:3743:50
     |
3743 |                     response_view: Option<crate::tasks::params::GetResponseView>,
     |                                                  ^^^^^
     |                                                  |
     |                                                  unresolved import
     |                                                  help: a similar path exists: `crate::projects::locations::queues::tasks`

error[E0433]: failed to resolve: unresolved import
    --> gen/cloudtasks/v2/lib/src/lib.rs:3760:39
     |
3760 |                         value: crate::tasks::params::GetResponseView,
     |                                       ^^^^^
     |                                       |
     |                                       unresolved import
     |                                       help: a similar path exists: `crate::projects::locations::queues::tasks`

error[E0433]: failed to resolve: unresolved import
    --> gen/cloudtasks/v2/lib/src/lib.rs:3885:50
     |
3885 |                     response_view: Option<crate::tasks::params::ListResponseView>,
     |                                                  ^^^^^
     |                                                  |
     |                                                  unresolved import
     |                                                  help: a similar path exists: `crate::projects::locations::queues::tasks`

error[E0433]: failed to resolve: unresolved import
    --> gen/cloudtasks/v2/lib/src/lib.rs:3912:39
     |
3912 |                         value: crate::tasks::params::ListResponseView,
     |                                       ^^^^^
     |                                       |
     |                                       unresolved import
     |                                       help: a similar path exists: `crate::projects::locations::queues::tasks`

error[E0423]: expected value, found struct `locations::LocationsActions`
    --> gen/cloudtasks/v2/lib/src/lib.rs:1065:13
     |
1065 |             locations::LocationsActions
     |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^ did you mean `locations::LocationsActions { /* fields */ }`?

error[E0423]: expected value, found struct `queues::QueuesActions`
    --> gen/cloudtasks/v2/lib/src/lib.rs:1118:17
     |
1118 |                 queues::QueuesActions
     |                 ^^^^^^^^^^^^^^^^^^^^^ did you mean `queues::QueuesActions { /* fields */ }`?

error[E0423]: expected value, found struct `tasks::TasksActions`
    --> gen/cloudtasks/v2/lib/src/lib.rs:1741:21
     |
1741 |                     tasks::TasksActions
     |                     ^^^^^^^^^^^^^^^^^^^ did you mean `tasks::TasksActions { /* fields */ }`?

warning: unreachable expression
    --> gen/cloudtasks/v2/lib/src/lib.rs:3312:29
     |
3312 | /                             Ok(match value {
3313 | |                                 _ => {
3314 | |                                     return Err(::serde::de::Error::custom(format!(
3315 | |                                         "invalid enum for #name: {}",
...    |
3318 | |                                 }
3319 | |                             })
     | |______________________________^
     |
     = note: #[warn(unreachable_code)] on by default

warning: unreachable expression
    --> gen/cloudtasks/v2/lib/src/lib.rs:3348:29
     |
3348 | /                             Ok(match value {
3349 | |                                 _ => {
3350 | |                                     return Err(::serde::de::Error::custom(format!(
3351 | |                                         "invalid enum for #name: {}",
...    |
3354 | |                                 }
3355 | |                             })
     | |______________________________^

error: aborting due to 7 previous errors

Some errors have detailed explanations: E0423, E0433.
For more information about an error, try `rustc --explain E0423`.
error: Could not compile `google-cloudtasks2`.

To learn more, run the command again with --verbose.
